addons/binding/org.openhab.binding.chamberlainmyq/README.md

```diff
@@ -0,0 +1,90 @@
+# Chamberlain MyQ Binding
+
+[Chamberlain MyQ](http://www.chamberlain.com/smartphone-control-products/myq-smartphone-control) system allows you to connect your garage door to the internet to be controlled from anywhere using your smartphone. Using this API, The Chamberlain MyQ Binding can get the status of your garage door opener and send commands to open or close it.
+
+## Supported Things
+
+```
+MyQ Gateway Thing Bridge - Gateway to MyQ Online API, Must be configured to use binding.
+MyQ Garage Door Thing - Standard Chamberlain Garage Door.
+MyQ Light Switch Thing - Chamberlain MyQ Controlled Light Switch or Plugin Module.
+```
+
+## Discovery
+
+Gararge Doors and Light Modules Things will be auto Discovery after a MYQ Gateway Thing is added manually.
+
+
+## Thing Configuration
+
+The Chamberlain MyQ Gateway Thing only requires your Chamberlain MyQ Username and Password.
+
+|   Property   | Default | Required | Description |
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍